README.md

```diff
@@ -16,22 +16,28 @@ A community repository for Helm Charts of OpenSearch Project.
 
 ## Status
 
-![Testing](https://github.com/opensearch-project/helm-charts/workflows/Lint%20and%20Test%20Charts/badge.svg)
-![Release](https://github.com/opensearch-project/helm-charts/workflows/Release%20Charts/badge.svg)
+[![Lint and Test Charts](https://github.com/opensearch-project/helm-charts/actions/workflows/lint-test.yaml/badge.svg)](https://github.com/opensearch-project/helm-charts/actions/workflows/lint-test.yaml)
+ [![Release Charts](https://github.com/opensearch-project/helm-charts/actions/workflows/release.yaml/badge.svg)](https://github.com/opensearch-project/helm-charts/actions/workflows/release.yaml)
 
 ## Installation
 
+To install the OpenSearch Helm charts, execute the following commands:
```

Comment:
As this is the central README and we have 2 charts in the repo as of now. Should we add some reference for our dashboards chart as well?

Reply:
Done